examples/logical_interconnect_groups.py

```diff
@@ -193,83 +197,74 @@
+ligs = logical_interconnect_groups.get_all(
     0, 10, sort='name:descending', filter="\"'name'='OneView Test Logical Interconnect Group'\"")
 pprint(ligs)
 
 # Get Logical Interconnect Group by property
-lig = oneview_client.logical_interconnect_groups.get_by('name', 'OneView Test Logical Interconnect Group')[0]
+lig = logical_interconnect_groups.get_by('name', 'SYN-LIG')[0]
 print("Found lig by name: '%s'.\n  uri = '%s'" % (lig['name'], lig['uri']))
 
+# Get Logical Interconnect Group by scope_uris
+if oneview_client.api_version >= 600:
+    lig_by_scope_uris = logical_interconnect_groups.get_all(scope_uris="\"'/rest/scopes/3bb0c754-fd38-45af-be8a-4d4419de06e9'\"")
+    if len(lig_by_scope_uris) > 0:
+        print("Found %d Logical Interconnect Groups" % (len(lig_by_scope_uris)))
+        i = 0
+        while i < len(lig_by_scope_uris):
+            print("Found Logical Interconnect Group by scope_uris: '%s'.\n  uri = '%s'" % (lig_by_scope_uris[i]['name'], lig_by_scope_uris[i]['uri']))
+            i += 1
+        pprint(lig_by_scope_uris)
+    else:
+        print("No Logical Interconnect Group found.")
+
+# Get logical interconnect group by name
+lig = logical_interconnect_groups.get_by_name(options["name"])
+if not lig:
+    # Create a logical interconnect group
+    print("Create a logical interconnect group")
+    lig = logical_interconnect_groups.create(options)
+    pprint(lig.data)
+
 # Update a logical interconnect group
 print("Update a logical interconnect group")
-lig_to_update = created_lig.copy()
+lig_to_update = lig.data.copy()
 lig_to_update["name"] = "Renamed Logical Interconnect Group"
-updated_lig = oneview_client.logical_interconnect_groups.update(lig_to_update)
-pprint(updated_lig)
-
-# Get all, with defaults
-print("Get all Logical Interconnect Groups")
-ligs = oneview_client.logical_interconnect_groups.get_all()
-pprint(ligs)
-
-# Get by Id
-try:
-    print("Get a Logical Interconnect Group by id")
-    lig_byid = oneview_client.logical_interconnect_groups.get('f0a0a113-ec97-41b4-83ce-d7c92b900e7c')
-    pprint(lig_byid)
-except HPOneViewException as e:
-    print(e.msg)
-
-# Get by uri
-try:
-    print("Get a Logical Interconnect Group by uri")
-    lig_byuri = oneview_client.logical_interconnect_groups.get(created_lig["uri"])
-    pprint(lig_byuri)
-except HPOneViewException as e:
-    print(e.msg)
+lig.update(lig_to_update)
+pprint(lig.data)
 
 # Performs a patch operation
 scope = oneview_client.scopes.get_by_name(scope_name)
 if scope:
     print("\nPatches the logical interconnect group adding one scope to it")
-    updated_lig = oneview_client.logical_interconnect_groups.patch(updated_lig['uri'],
-                                                                   'replace',
-                                                                   '/scopeUris',
-                                                                   [scope['uri']])
-    pprint(updated_lig)
+    updated_lig = lig.patch('replace',
```

Comment:
i think this endpoint has been removed from API600 onwards. Lets add a check here.

Reply:
Done